packages/app-webdir-ui/src/ProfileCard/index.js

```diff
@@ -89,7 +89,7 @@ const ProfileCard = ({ ...props }) => {
               <a
                 onClick={() => sendEvent(props.email)}
                 href={`mailto:${props.email.toLowerCase()}`}
-                aria-label="Email user"
+                aria-label={`Mail to :${props.email.toLowerCase()}`}
```

Comment:
We are no longer casting email addresses to lower case. Can you remove the `.toLowerCase()` ?